cli/command/node/client_test.go

```diff
@@ -0,0 +1,85 @@
+package node
+
+import (
+	"io"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/swarm"
+	"github.com/docker/docker/cli/command"
+	"github.com/docker/docker/client"
+	"golang.org/x/net/context"
+)
+
+type fakeCli struct {
+	command.DockerCli
+	client client.APIClient
+	out    io.Writer
+}
```

Comment:
This could definitely be used by unit tests in other packages.

What do you think about using https://github.com/vektra/mockery to generate something like this instead of writing them manually?